### PR TITLE
Remove tailing semi-colons in .swift

### PR DESCRIPTION
**Pull request by dduan**
The sed Force Awakens (sorry, did this while waiting for the movie to start).

`build-script -RT` all passes on OS X.

### CODE REVIEW

File: test/Parse/toplevel_library.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // RUN: %target-parse-verify-swift -parse-as-library
 
 // make sure trailing semicolons are valid syntax in toplevel library code.
-var x = 4;
+var x = 4
```
Comment (gribozavr) [inline test/Parse/toplevel_library.swift]: This one needs to stay.

**Comment by gribozavr** [comment]
Well, if we remove all of these semicolons (which I agree is the right thing for individual tests), we decrease our test coverage for semicolons.  Would you mind writing some targeted tests for the parser?

**Comment by dduan** [comment]
@gribozavr I'll take a stab at it.

**Comment by dduan** [comment]
@gribozavr test added, what you think?

**Comment by gribozavr** [comment]
@dduan Thanks!

**Pull request merged**